### PR TITLE
feat: add gradient generator by josh comeau

### PR DESCRIPTION
**Pull request by YayoKB**

### CODE REVIEW

File: src/data/tools.json [hunk 1]
```diff
@@ -1290,82 +1325,98 @@
...
+    "tag": [
+      "css",
+      "tool",
+      "tools",
+      "design",
+      "generator",
+      "generators",
+      "utility",
+      "utilities",
+      "online tools"
```
Comment (schalkneethling) [inline src/data/tools.json:1419]: I wonder whether we should add gradient generator?

**Review by schalkneethling — COMMENTED** [review]
Thank you so much for your contribution, @YayoKB 🎉 Only have one suggestion.

File: src/data/tools.json (hunk 1 above)
Comment (YayoKB) [reply src/data/tools.json:1419]: I think that would be a good idea because it would work if the tool chest grows :)

Comment (YayoKB) [reply src/data/tools.json]: Added the tag!

**Pull request merged**